builder/builder.go

```diff
@@ -191,19 +187,6 @@ func BuildReplaceInsert(table string, data []map[string]interface{}) (string, []
 	return buildInsert(table, data, replaceInsert)
 }
 
-var (
-	cpPool = sync.Pool{
-		New: func() interface{} {
-			return make([]Comparable, 0)
-		},
-	}
-)
-
-func getCpPool() ([]Comparable, func()) {
-	obj := cpPool.Get().([]Comparable)
-	return obj[:0], func() { cpPool.Put(obj) }
-}
-
 func emptyFunc() {}
```

Comment:
this line also need to be removed?

Reply:
Yes, emptyFunc is meant for satisfy the function signature, and it's useless now